set.mm

```diff
@@ -55107,8 +55108,10 @@ the restriction (of the relation) to the singleton containing this
 
   ${
     $d x y A $.  $d x y B $.  $d x y C $.
-    $( Subset of the range of a restriction.  (Contributed by NM, 16-Jan-2006.)
-       (Proof shortened by Peter Mazsa, 2-Oct-2022.) $)
+    $( Two ways to express surjectivity of a restricted and corestricted binary
+       relation (intersection of a binary relation with a Cartesian product).
+       (Contributed by NM, 16-Jan-2006.)  (Proof shortened by Peter Mazsa,
+       2-Oct-2022.) $)
```

Comment:
The former "Subset of the range of a restriction. " describes precisely the LHS of the statement. Therefore it should remain in the comment (somehow). The term "corestricted" is not used/defined in set.mm yet.

Reply:
Now I see:  "restricted and corestricted binary relation" means "intersection of a binary relation with a Cartesian product".

Reply:
I added an explanation in the comment of df-res, and I'll add a link when I add the theorem
```
cnvrescnv $p |- `' ( `' R |` B ) = ( R i^i ( _V X. B ) ) $= ? $.
```

Reply:
(done)